client/src/pages/CreateProduct.js

```diff
@@ -60,70 +68,164 @@ const typeOfProduct = [
+              Product Created!
+            </Alert>
+          </Snackbar>
+          <Snackbar open={errorOpen} autoHideDuration={6000} onClose={handleClose}>
+            <Alert onClose={handleClose} severity="error">
+              Product Already Exists!
+            </Alert>
+          </Snackbar>
+        </div>
+        <Grid container spacing={1}>
+          <Grid item xs={6}>
+            <h1 className={classes.marginTop136}>Upload new product</h1>
 
-                <div className={classes.rootPaper}>
-                  <Paper elevation={3} />
-                  <Paper elevation={3} />
-                  <Paper elevation={3} />
-                </div>
-                <div className={classes.rootPaper}>
-                  <Paper elevation={3} />
-                  <Paper elevation={3} />
-                  <Paper elevation={3} />
-                </div>
-              </Grid>
-              <Grid item xs={6}>
-                <form className={classes.rootForm}>
-                  <TextField id="outlined-basic" label="Title" variant="outlined" fullWidth />
-                  <TextField id="outlined-basic" label="Description" variant="outlined" fullWidth rows={4} multiline/>
-                  <Grid container spacing={1}>
-                    <Grid item xs={3}>
-                      <FormControl variant="outlined" fullWidth>
-                        <InputLabel htmlFor="outlined-adornment-amount">Amount</InputLabel>
-                        <OutlinedInput
-                          id="outlined-adornment-amount"
-                          label="Amount"
-                          startAdornment={<InputAdornment position="start">$</InputAdornment>}
-                        />
-                      </FormControl>
-                    </Grid>
-                    <Grid item xs={9}>
-                      <TextField
-                        id="outlined-select-currency-native"
-                        select
-                        fullWidth
-                        label="Type of Product"
-                        SelectProps={{
-                          native: true,
-                        }}
-                        variant="outlined"
-                      >
-                        {typeOfProduct.map((option) => (
-                          <option key={option.value} value={option.value}>
-                            {option.label}
-                          </option>
-                        ))}
-                      </TextField>
-                    </Grid>
-                  </Grid>
-                </form>
-              </Grid>
-            </Grid>
-            <Grid className={classes.marginTop64} container justify="center">
-              <Grid item xs={4}>
-                <Button variant="outlined" fullWidth size="large">Upload</Button>
+            <div className={classes.rootPaper}>
+              <Paper elevation={3}>
+                <input id="1" type="file" onChange={handlePhotoButton} />
```

Comment:
i wonder if you can map all these `<Paper>` and `<input`> since the only difference is the `id` 

something similar to 

```
["1", "2", "3", "4", "5", "6"].map(num => (
  <Paper elevation={3}>
    <input id={num} type="file" onChange={handlePhotoButton} />
  </Paper>
))
```